worker/awsCertlint/awsCertlint.go

```diff
@@ -0,0 +1,167 @@
+package awsCertlint
+
+import (
+	"bufio"
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"strings"
+
+	"github.com/mozilla/tls-observatory/certificate"
+	"github.com/mozilla/tls-observatory/worker"
+)
+
+var (
+	workerName = "awsCertlint"
+	workerDesc = "Runs awslabs/certlint over a given Certificate, categoriezes output for display on the certificate"
```

Comment:
s/categoriezes/categorize/, but even though that sentence doesn't make a lot of sense. can you rephrase?